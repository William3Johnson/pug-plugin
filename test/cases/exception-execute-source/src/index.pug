html
  head
    link(rel='stylesheet' href=require('./not-found.css'))
  body
    p Hello World!