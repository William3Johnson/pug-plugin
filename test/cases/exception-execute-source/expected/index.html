<!DOCTYPE html><html><head><link rel="stylesheet" href="/assets/css/main.816f57966f4d0112e799.css"><link rel="stylesheet" href="/assets/css/about.6f89299362d6b4ff1776.css"></head><body><p>The usage of the style source directly in the pug template.</p></body></html>